general_superstaq/superstaq_client.py

```diff
@@ -304,6 +304,19 @@ def ibmq_set_token(  # pylint: disable=missing-param-doc,differing-param-doc
 
         return self.post_request("/ibmq_token", json_dict)
 
+    def cq_set_token(  # pylint: disable=missing-param-doc,differing-param-doc
```

Comment:
are these pragmas necessary? if not:
```suggestion
    def cq_set_token(
```